actionpack/lib/action_controller/metal/conditional_get.rb

```diff
@@ -77,18 +77,16 @@ def etag(&etagger)
     #
     #   before_action { fresh_when @article, template: 'widgets/show' }
     #
-    def fresh_when(record_or_options, additional_options = {})
-      if record_or_options.is_a? Hash
```

Comment:
The old implementation suggests I could pass in directly a hash instead of a record, even thought there are no docs about it, there might be people using it that way.

Will this still work?